### ISSUE TITLE
False positive on 'Page must have one main landmark'?

### ISSUE THREAD

**Issue by mrshllstock** [dequelabs/axe-core#3416]
Product: axe Extension

Expectation: Not report '[Page must have one main landmark](https://dequeuniversity.com/rules/axe/4.4/landmark-one-main?application=AxeChrome)' for dialogs where the main landmark has aria-hidden='true'.

Actual: Reports 'Page must have one main landmark' for dialogs where the main landmark has aria-hidden='true'.

Motivation: Reduce false-positives of acceptable behaviour from an A11y point of view. With JAWS and NVDA, depending on the location of the virtual cursor, navigation behaviour changes pending the content behind being aria-hidden.

With aria-hidden: cannot navigate to content behind the dialog with shortcuts such as CTRL + Home or CTRL + End.

Without aria-hidden: Can navigate to content behind the dialog though means such as clicking to move the focus to the top of the doc or CTRL + F. Similarly once the virtual cursor was outside of the dialog, navigation became restricted to outside of the dialog.

This seems to be more desirable behaviour to have the content not in the dialog to be aria-hidden.

<pre><code>
axe-webdriver, extension or other integration version: 4.24.4
</code></pre>

**Comment by straker** [comment]
Thanks for the issue. I believe this is a duplicate of https://github.com/dequelabs/axe-core/issues/3359. Going to close, but feel free to reopen if you feel that is not the case.
